packages/report-server/src/reportBuilder/transform/functions/fillRows/fillRows.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { yup } from '@tupaia/utils';
+
+import { FieldValue, Row } from '../../../types';
+import { Context } from '../../../context';
+import { TransformParser } from '../../parser';
+import { TransformTable } from '../../table';
+import { checkColumnsExistInTable, expressionOrArrayValidator } from '../utils';
+import { generateRowInserts } from './generateRowInserts';
+
+type FillRowsParams = {
+  requiredColumnValues: { column: string; values: string | FieldValue[] }[];
+  missingRowValues: Row;
```

Comment:
Arguably this field should be called `columns` instead of `missingRowValues` to be consistent with `insertRows`.
However, I think that's pretty confusing in this context, and I don't really feel it's too intuitive in `insertRows` either...

I've got Nikhita and Gagan having a play with this transform and I'll get their feedback on the naming of the fields.